plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSplitManager.java

```diff
@@ -93,20 +94,44 @@ public void testInitialSplits()
         List<DeltaLakeSplit> splits = getSplits(splitManager, deltaLakeConfig);
 
         List<DeltaLakeSplit> expected = ImmutableList.of(
-                makeSplit(0, 5_000, fileSize, minimumAssignedSplitWeight),
-                makeSplit(5_000, 5_000, fileSize, minimumAssignedSplitWeight),
-                makeSplit(10_000, 5_000, fileSize, minimumAssignedSplitWeight),
-                makeSplit(15_000, 5_000, fileSize, minimumAssignedSplitWeight));
+                makeSplit(FULL_PATH, 0, 5_000, fileSize, minimumAssignedSplitWeight),
+                makeSplit(FULL_PATH, 5_000, 5_000, fileSize, minimumAssignedSplitWeight),
+                makeSplit(FULL_PATH, 10_000, 5_000, fileSize, minimumAssignedSplitWeight),
+                makeSplit(FULL_PATH, 15_000, 5_000, fileSize, minimumAssignedSplitWeight));
 
         assertEquals(splits, expected);
     }
 
+    @Test
+    public void testAbsolutePathSplits()
```

Comment:
could you please add a `dataProvider` here and test the splitting for several object storages ? (s3, abfs, at the very least)

Reply:
@findinpath added the cases for `s3, abfs, hdfs, gs, file`